htdocs/accountancy/class/accountancyimport.class.php

```diff
@@ -55,21 +85,26 @@ public function computeAmount(&$arrayrecord, $fieldname, &$listfields, &$listval
 			$fieldname = $fieldArr[1];
 		}
 
-		$debit  = floatval(trim($arrayrecord[11]['val']));
-		$credit = floatval(trim($arrayrecord[12]['val']));
+		$debit_index = 11;
```

Comment:
This method is used by the import process. The field debit will not always be at position 11.